lib/Test.js

```diff
@@ -41,6 +41,7 @@ define([
 		 * @see Suite#remote
 		 */
 		get remote() {
+			this._usesRemote = true;
```

Comment:
The flag needs to be reset when the test is reset. It should also be listed explicitly in the prototype with its default value.